ui/packages/catalog-ui-search/src/main/webapp/react-component/layer-item/presentation/name.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+import { hot } from 'react-hot-loader'
+import { PresentationProps, DisabledBehavior } from '.'
+
+const Name = styled.div`
+  line-height: ${props => props.theme.minimumButtonSize};
+  overflow: hidden;
+  text-overflow: ellipsis;
+`
+const NameDisabled = styled(Name)`
+  ${props => DisabledBehavior(props.theme)};
+  cursor: text !important;
```

Comment:
:question: this doesn't work without the `important` tag?

Reply:
The DisabledBehavior mixin I used has the css property `cursor: not-allowed !important`, so I have to override that here.